src/audiovideocontroller/AudioVideoController.ts

```diff
@@ -34,6 +34,15 @@ export default interface AudioVideoController extends AudioVideoControllerFacade
    */
   restartLocalVideo(callback: () => void): void;
 
+  /**
+   * Replaces the local video device without a WebRTC negotiation on the sender side and
+   * avoids that all video subscribers to re-negotiate at the same time.
+   * This is an optimized way of replacing video stream and requires `WebRTC Unified Plan`.
+   * See {@link MeetingSessionConfiguration.enableUnifiedPlanForChromiumBasedBrowsers} and
+   * {@link BrowserBehavior.requiresUnifiedPlan}.
+   */
+  replaceLocalVideo?(callback: () => void): Promise<void>;
```

Comment:
It is insane to both return a `Promise` and take a callback. Do one or the other. I recommend killing the callback.

Reply:
fixed.